src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -14,10 +14,89 @@
 
 package codeu.controller;
 
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 /** Servlet class responsible for the chat page. */
 public class AdminServlet extends HttpServlet {
-  // TODO(JW): To be implemented.
 
+  /** Store class that gives access to Conversations. */
+  private static ConversationStore conversationStore;
```

Comment:
So, I don't think you want these to be be static actually.  Take a look at one of the existing Servlet implementations so see what they do.  Honestly I'm not 100% sure what the reason is for doing it one way or another, since the Store implementations are all singletons, but I figure we should stick to the pattern.  (It might have something to do with having multiple threads... but not sure.)

Reply:
I see, I agree it's better to stick to the pattern, fixed